docs/modules/ROOT/pages/keyboard-shortcuts.adoc

```diff
@@ -0,0 +1,55 @@
+= Keyboard Shortcuts
```

Comment:
I think we can skip the shortcuts, wdyt @yuce

Reply:
IMO it's nice to have them in the docs.